index.js

```diff
@@ -62,9 +56,16 @@ GitHub.prototype = {
       cb = data;
       data = null;
     }
+
+    cb = typeof cb === 'function' ? cb : function noop () {};
```

Comment:
Yea it's possible, but it adds need to do test for it. I just trying to not do `if` statements for such things, because the coverage.